torch/distributed/pipelining/stage.py

```diff
@@ -111,6 +111,10 @@ def _make_tensor_from_meta(
     )
 
 
+def _rank_to_global_rank(peer_rank: int, group: Optional[dist.ProcessGroup]):
```

Comment:
i think you can just directly use this helper from distributed_c10d.py?

```
def get_global_rank(group: ProcessGroup, group_rank: int) -> int:
    """
    Translate a group rank into a global rank.

    ``group_rank`` must be part of `group` otherwise this raises RuntimeError.

    Args:
        group (ProcessGroup): ProcessGroup to find the global rank from.
        group_rank (int): Group rank to query.

    Returns:
        Global rank of ``group_rank`` relative to ``group``

    N.B. calling this function on the default process group returns identity
    """
```

Reply:
oops yeah, good point!